drivers/i2c/i2c_npcx_controller.c

```diff
@@ -1170,6 +1200,26 @@ int npcx_i2c_ctrl_recover_bus(const struct device *dev)
 }
 
 #ifdef CONFIG_I2C_TARGET
+static volatile uint8_t *npcx_i2c_ctrl_target_get_reg_smbaddr(const struct device *i2c_dev,
+							      int index)
+{
+	struct smb_reg *const inst = HAL_I2C_INSTANCE(i2c_dev);
+
+	switch (index) {
+	case 0: return &inst->SMBADDR1;
```

Comment:
```suggestion
	case 0:
		return &inst->SMBADDR1;
```

and so on, sorry I know it makes this fairly long but the general coding style typically avoids patterns like this

Reply:
Done